go/worker/keymanager/worker.go

```diff
@@ -287,6 +302,11 @@ func (w *Worker) updateStatus(status *api.Status, startedEvent *host.StartedEven
 	w.logger.Info("Key manager initialized",
 		"checksum", hex.EncodeToString(signedInitResp.InitResponse.Checksum),
 	)
+	if w.initTicker != nil {
+		w.initTickerCh = make(chan time.Time)
```

Comment:
Why not set the channel to `nil`, it should always block then (e.g., select case should never trigger).

Reply:
:+1: also fixed the init code to initialize it to a nil channel.